### ISSUE TITLE
Absolute date ranges for event time window

### ISSUE THREAD

**Issue by sintezcs** [dittofeed/dittofeed#991]
Right now you can only specify relative time window, like `User Performed test_event Withing The Last 5 Days`

For marketing scenarios, absolute time windows can be really beneficial.
A good example of flexible time window is laudspeakers implementation:
![Screen Recording 2024-08-22 at 20 04 46-3](https://github.com/user-attachments/assets/c6cec7a7-acd0-46d5-857b-33cf9c752f3c)

**Comment by maxgurewitz** [comment]
great suggestion!

**Comment by ananthu-kurup** [comment]
Would like to pick this up, @maxgurewitz

**Comment by maxgurewitz** [comment]
@ananthu-kurup feel free to give it a shot, it is a more challenging issue though!

**Comment by ananthu-kurup** [comment]
> @ananthu-kurup feel free to give it a shot, it is a more challenging issue though!

Yeah, I will need some support in understanding the backend code once I start. But one of the reasons I picked this up is the depth of the problem itself seems exciting and challenging.

**Comment by maxgurewitz** [comment]
Hi @titvilasi [v0.23.0-alpha.77](https://github.com/dittofeed/dittofeed/releases/tag/v0.23.0-alpha.77) now includes absolute date range filtering within Trait segments! This feature as applied to Performed segments will come later!
